<ui version="4.0" >
 <author>Andrey Rijov</author>
 <class>Convert_HDD_Image_Window</class>
 <widget class="QDialog" name="Convert_HDD_Image_Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Convert HDD Image</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/hdd.png</normaloff>:/hdd.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="Label_Base_File_Name" >
     <property name="text" >
      <string>&amp;Input Image File Name:</string>
     </property>
     <property name="buddy" >
      <cstring>Edit_Base_File_Name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="Edit_Base_File_Name" />
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="Button_Browse_Base" >
     <property name="text" >
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="Label_Input_Format" >
     <property name="text" >
      <string>Input &amp;Format:</string>
     </property>
     <property name="buddy" >
      <cstring>CB_Input_Format</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QComboBox" name="CB_Input_Format" >
     <property name="maxCount" >
      <number>32</number>
     </property>
     <item>
      <property name="text" >
       <string>qcow2</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>qcow</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>vmdk</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>cow</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>raw</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>cloop</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="Button_Identify" >
     <property name="text" >
      <string>I&amp;dentify</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="Label_Output_File_Name" >
     <property name="text" >
      <string>&amp;Output Image File Name:</string>
     </property>
     <property name="buddy" >
      <cstring>Edit_Output_File_Name</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLineEdit" name="Edit_Output_File_Name" />
   </item>
   <item row="5" column="1" >
    <widget class="QPushButton" name="Button_Browse_Output" >
     <property name="text" >
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="Label_Output_Format" >
     <property name="text" >
      <string>Ou&amp;tput Format:</string>
     </property>
     <property name="buddy" >
      <cstring>CB_Output_Format</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QComboBox" name="CB_Output_Format" >
     <property name="maxCount" >
      <number>32</number>
     </property>
     <item>
      <property name="text" >
       <string>qcow2</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>qcow</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>vmdk</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>cow</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>raw</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>cloop</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>200</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QGroupBox" name="GB_QCOW_Options" >
     <property name="title" >
      <string>QCOW Format Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="CH_Compressed" >
        <property name="text" >
         <string>Image Must be Com&amp;pressed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="CH_Encrypted" >
        <property name="text" >
         <string>Image Must be &amp;Encrypted</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>141</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Convert" >
       <property name="text" >
        <string>Con&amp;vert</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/go.png</normaloff>:/go.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <tabstops>
  <tabstop>Edit_Base_File_Name</tabstop>
  <tabstop>Button_Browse_Base</tabstop>
  <tabstop>CB_Input_Format</tabstop>
  <tabstop>Button_Identify</tabstop>
  <tabstop>Edit_Output_File_Name</tabstop>
  <tabstop>Button_Browse_Output</tabstop>
  <tabstop>CB_Output_Format</tabstop>
  <tabstop>CH_Compressed</tabstop>
  <tabstop>CH_Encrypted</tabstop>
  <tabstop>Button_Convert</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Convert_HDD_Image_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>457</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
